packages/hash/api/src/model/knowledge/link.model.ts

```diff
@@ -134,5 +305,31 @@ export default class {
       targetEntityId: this.targetEntityModel.entityId,
       removedById: removedBy,
     });
+
+    if (this.index !== undefined) {
+      const siblingLinks = await this.sourceEntityModel.getOutgoingLinks(
+        graphApi,
+        {
+          linkTypeModel: this.linkTypeModel,
+        },
+      );
+
+      const affectedSiblings = siblingLinks.filter(
+        (sibling) => sibling.index! >= this.index!,
```

Comment:
Shouldn't be equal to right?

Reply:
I think that since we're first removing the current link, the siblings list would not contain the link that would be equal to, and no extra update would be issued. but yes `>` instead may make more sense

Reply:
Good spot, thank you - you're right it should be `sibling.index! > this.index! here. Have addressed in https://github.com/hashintel/hash/pull/1081/commits/9c37a55c638a722b8057f6b68440c18f27812168